### PR TITLE
added estate agent request class in models, added /request/estate-age…

### PR DESCRIPTION
**Pull request by theobaur13**
…nt, added EstateAgentRequestForm and RequestHandleForm in forms, added handling of estate agent requests in admin-dashboard

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -75,6 +77,19 @@ class Report(db.Model):
...
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    estate_agent_id = db.Column(db.Integer, db.ForeignKey('estate_agent.id'), nullable=True)
+    name = db.Column(db.String(256), nullable=False)
+    email = db.Column(db.String(256), nullable=False)
+    phone = db.Column(db.String(256), nullable=False)
+    website = db.Column(db.String(256), nullable=False)
+    date = db.Column(db.DateTime, nullable=False, default=datetime.now())
+    handled = db.Column(db.Boolean, nullable=False, default=False)
+    user = db.relationship('User', back_populates='estate_agent_requests', lazy=True)
+    estate_agent = db.relationship('EstateAgent', back_populates='requests', lazy=True)
```
Comment (tobynott80) [inline app/models.py:91]: Why would an estate agent request have a relationship with an existing estate agent? or am i just being stupid haha

**Review by tobynott80 — APPROVED** [review]
LGTM 🫡

File: app/models.py (hunk 1 above)
Comment (theobaur13) [reply app/models.py:91]: When the request is approved a new estate agent gets added and the original request is modified so it links to the new estate agent, not essential but could be useful to retrieve the linked estate agent from the request

Comment (tobynott80) [reply app/models.py:91]: Ah nice one, makes sense

**Pull request merged**